BrowserKit/Sources/OnboardingKit/Views/OnboardingView/OnboardingView.swift

```diff
@@ -0,0 +1,55 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import SwiftUI
+
+// MARK: - Main Onboarding View
+public struct OnboardingView<VM: OnboardingCardInfoModelProtocol>: View {
+    @Environment(\.horizontalSizeClass)
+    private var horizontalSizeClass
+    @StateObject private var viewModel: OnboardingFlowViewModel<VM>
+    let windowUUID: WindowUUID
+    var themeManager: ThemeManager
+
+    public init(
+        windowUUID: WindowUUID,
+        themeManager: ThemeManager,
+        viewModel: OnboardingFlowViewModel<VM>
+    ) {
+        self.windowUUID = windowUUID
+        self.themeManager = themeManager
+        _viewModel = StateObject(
+            wrappedValue: viewModel
+        )
+    }
+
+    public var body: some View {
+        Group {
+            if horizontalSizeClass == .regular {
```

Comment:
With this we also would display the regular view on iPhone in landscape - do we want that. Especially since compact and regular have a different amount of cards. Maybe we need to add the cards based on device but display the UI based on size classes? I am ok if this is addressed in another ticket.